css/onoffice-forms.css

```diff
@@ -9,4 +9,8 @@ span.onoffice-pleasefill {
 	opacity: 0.5;
 	cursor: not-allowed;
 	pointer-events: none;
+}
+
+.oo-form .hidden-field {
+	display: none;
```

Comment:
@andernath  We should keep the parent class "oo-form" because some themes will have their own CSS styles to customize input fields. If you don't use the parent class, the priority of the above style snippet will be lower, and then the attribute "display:none" will not be executed.